language/move-lang/src/shared/mod.rs

```diff
@@ -58,8 +58,9 @@ impl Address {
         assert!(result.len() >= 32);
         Self::try_from(&result[..]).map_err(|_| {
             format!(
-                "The address {:?} is of invalid length. Addresses are at most 32-bytes long",
-                result
+                "Address is {} bytes long; the maximum size is {} bytes",
```

Comment:
super nit. I tend to just use a period (over a semi-colon) for error messages like this

Reply:
OK